tests/pre-exit-success.bats

```diff
@@ -57,6 +57,26 @@ COMMON_CURL_OPTIONS='--form \* --form \* --form \* --form \* --form \* --form \*
   assert_output --partial "curl success 3"
 }
 
+@test "Uploads multiple files concurrently" {
+  export BUILDKITE_PLUGIN_TEST_COLLECTOR_FILES='**/*/junit-*.xml'
+  export BUILDKITE_PLUGIN_TEST_COLLECTOR_UPLOAD_CONCURRENCY='3'
+
+  stub curl \
+    "-X POST --silent --show-error --max-time 30 --form format=junit ${COMMON_CURL_OPTIONS} \* -H \* : echo 'curl success 1'"
```

Comment:
I would rather you tailored the output, something like:
```suggestion
    "-X POST --silent --show-error --max-time 30 --form format=junit ${COMMON_CURL_OPTIONS} \* -H \* : echo curl success \${10};"
```

Reply:
Thanks @toote, I'll have a play around with that, I was struggling to bend the bats tests to my will... but I'll give it another go on monday.

Reply:
this was supposed to be part of my review but I obviously clicked the wrong button, sorry. But feel free to reach out to me and I'll be more than happy to assist with you fighting against bats (I know I've had my share of rounds with it) :)